source/nodejs/adaptivecards-designer/src/containers/host-container.ts

```diff
@@ -160,6 +245,63 @@ export abstract class HostContainer {
         });
     }
 
+    protected renderSpeech(speechString: string, showXml: boolean = false): HTMLElement {
```

Comment:
There is no "Read this card" button in the Designer's UI, therefore this isn't necessary.

Reply:
Don't change - we'll add the "read this card" button later ourselves, so code can stay as is